src/CMakeLists.txt

```diff
@@ -136,3 +136,19 @@ ign_build_tests(TYPE UNIT SOURCES ${gtest_sources} LIB_DEPS ${rendering_target})
 ign_build_tests(TYPE UNIT SOURCES Lidar_TEST.cc LIB_DEPS ${lidar_target})
 ign_build_tests(TYPE UNIT SOURCES Camera_TEST.cc LIB_DEPS ${camera_target})
 ign_build_tests(TYPE UNIT SOURCES ImuSensor_TEST.cc LIB_DEPS ${imu_target})
+
+set(plugin_test_targets)
+list(APPEND plugin_test_targets "UNIT_Camera_TEST")
+list(APPEND plugin_test_targets "UNIT_Lidar_TEST")
+list(APPEND plugin_test_targets "UNIT_ImuSensor_TEST")
```

Comment:
We could miss adding other tests to this list here. How about creating a list with `Lidar`, `Camera` and `ImuSensor` and using that in a loop to build the tests (lines 136-138), as well as set the env vars?

Reply:
these tests are treated separately because they each depend on different components / plugins. I was able to make a loop for setting the environment variables because the paths are the same in each case, but the `LIB_DEPS` are different

Reply:
Got it. I think we could play with the target names to make the loop possible, but I realize now that it's not as simple as I originally thought.